common/app/controllers/EmailSignupController.scala

```diff
@@ -260,6 +270,7 @@ class EmailSignupController(
               s"email: ${form.email}, " +
               s"ref: ${form.ref}, " +
               s"refViewId: ${form.refViewId}, " +
+              s"g-recaptcha-response: ${form.googleRecaptchaResponse}, " +
```

Comment:
I'm assuming we get here when after submitting the form (controlled by newsletters.js)? If yes, is this where we'd extend to validate the captcha token (something I thought I read in the PR description as a follow-on PR)? And, out of curiosity, what validation will we do and how would that inform/change the flow from what we have now?

Reply:
that's correct - we reach here when an email signup form is submitted, either from `newsletters.js` or `common/app/views/emailEmbed.scala.html`. In another PR, I will implement validating this token with a call to the Google API (https://developers.google.com/recaptcha/docs/verify).

Although not finalised, if the validation fails (due to a stale or re-used token), we would probably return a `500` - which is the current behaviour if Identity rejects the request due to rate limiting.